brainscore_vision/benchmarks/imagenet_c/benchmark.py

```diff
@@ -82,10 +82,10 @@ def __init__(self, noise_category, sampling_factor=10):
         self.noise_types = self.noise_category_map[noise_category]
 
         ceiling = Score(1)
-        super(Imagenet_C_Category, self).__init__(identifier=f'dietterich.Hendrycks2019-{noise_category}-top1',
+        super(Imagenet_C_Category, self).__init__(identifier=f'Hendrycks2019-{noise_category}-top1',
```

Comment:
in case you'd like to update the ML benchmark identifiers, this is another one of those

Reply:
Updated ML benchmarks in recent commits.